provider/marathon.go

```diff
@@ -427,7 +426,9 @@ func (provider *Marathon) getFrontendBackend(application marathon.Application) s
 func (provider *Marathon) getSubDomain(name string) string {
 	if provider.GroupsAsSubDomains {
 		splitedName := strings.Split(strings.TrimPrefix(name, "/"), "/")
-		sort.Sort(sort.Reverse(sort.StringSlice(splitedName)))
+		for i, j := 0, len(splitedName)-1; i < j; i, j = i+1, j-1 {
+			splitedName[i], splitedName[j] = splitedName[j], splitedName[i]
+		}
```

Comment:
Sure. Reverse's `Less` implementation just reverses the `Less` implementation of the underlying type. So when we call Sort, it still returns in sorted order. If you have a suggested way to do this using the stdlib, I'd be happy to make that change.